specification/08-attestation.adoc

```diff
@@ -1,2 +1,94 @@
 [[attestation]]
 == Device Attestation
+
+There are two attestation involved in the CoVE-IO architecture.
+- TVM attests the Device
+- 3rd party attests the TVM with the device
+
+The IETF RATS Remote Attestation Architecture <<RATS>> can be applied to both.
+
+.Remote Attestation Framework (IETF RATS)
+image::images/rats-arch.svg[align="center"]
+
+=== TVM attesting the Device
+
+The TVM needs to verify and trust the device before offload the workload to the
+TDI. According to <<TDISP>> architecture, the high level steps are as follows:
+
+1. The TSM collects the device certificate and measurement from the DSM as the
+   evidence, via SPDM protocol.
+2. The TSM presents the evidence to the TVM.
+3. A TVM verifier appraises the evidence and produce the attestation result to
+   the relying party in the TVM, such as device consumer.
```

Comment:
What do you mean by "such as device consumer" ?

Reply:
I mean the code that uses the device, such as vendor's device driver, or generic PCI driver.